YoYoProject/GMProject.cs

```diff
@@ -88,7 +89,14 @@ internal override void Deserialize(ModelBase model)
             JavaScript = project.option_ecma;
 
             Parent.Deserialize(project.parentProject);
-            Resources.Deserialize(project.resources);
+            try
+            {
+                Resources.Deserialize(project.resources);
+            }
+            catch (InvalidOperationException e)
```

Comment:
Inappropriate squashing of an exception in the library. This needs to be done at a higher level, we should not make the assumption for how the user of the library wants to deal with exceptions.